include/effects/matrix/PatternWeather.h

```diff
@@ -81,7 +101,16 @@ static std::map<int, EmbeddedFile, std::less<int>, psram_allocator<std::pair<int
     { 10, EmbeddedFile(rain_start, rain_end) },
     { 11, EmbeddedFile(thunderstorm_start, thunderstorm_end) },
     { 13, EmbeddedFile(snow_start, snow_end) },
-    { 50, EmbeddedFile(mist_start, mist_end) }
+    { 50, EmbeddedFile(mist_start, mist_end) },
+    { 101, EmbeddedFile(clearsky_night_start, clearsky_night_end) },
+    { 102, EmbeddedFile(fewclouds_night_start, fewclouds_night_end) },
+    { 103, EmbeddedFile(scatteredclouds_night_start, scatteredclouds_night_end) },
+    { 104, EmbeddedFile(brokenclouds_night_start, brokenclouds_night_end) },
+    { 109, EmbeddedFile(showerrain_night_start, showerrain_night_end) },
+    { 110, EmbeddedFile(rain_night_start, rain_night_end) },
+    { 111, EmbeddedFile(thunderstorm_night_start, thunderstorm_night_end) },
+    { 113, EmbeddedFile(snow_night_start, snow_night_end) },
+    { 150, EmbeddedFile(mist_night_start, mist_night_end) }

```

Comment:
Agreed.  I had left it as it was because I did not fully understand the map template.  I read up on it and now it is properly using the actual icon string from the Weather API.